app/lib/bk/compat/parsers/circleci/steps.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
 require_relative 'logic'
-require_relative '../../pipeline/step'
```

Comment:
This file still uses the circlecci-specific step class so it should include the dependencies explicitly (remember to update the path if the file is renamed or moved as suggested):

```suggestion
require_relative '../../models/steps/circleci/circleci_step'
```

Reply:
Adjusted - also adjusted the step creation to `BK::Compat::CircleCIStep` - fully qualified class name on line 112 👍